spec.bs

```diff
@@ -937,10 +949,13 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
 
   <dl dfn-for="shared storage page budget">
     : <dfn>overall budget</dfn>
-    :: a double
+    :: a {{double}}
 
     : <dfn>site budget map</dfn>
-    :: a [=map=] of [=sites=] to doubles
+    :: a [=map=] of [=site=] to {{double}}
+
+    : <dfn>saved query map</dfn>
+    :: a [=map=] of [=tuples=] ([=url/origin=], [=/URL=], [=string=], [=string=]) to {{unsigned long}}
```

Comment:
I recommend phrasing this like `a [=string=] <dfn for="saved query map">operation name</dfn>, a [=string=] <dfn for="saved query map">saved query name< /dfn>`, etc.  So folks can tell the difference between the strings, etc.  I would do it for the URL and origin as well for completeness.

Reply:
Done.